packages/common/src/sites/HubSites.ts

```diff
@@ -489,18 +489,30 @@ export async function convertItemToSite(
  * @returns
  */
 export async function searchSites(
-  filter: Filter<"content">,
+  query: string | IQuery,
   options: IHubSearchOptions
-): Promise<ISearchResponse<IHubSite>> {
-  // Scope to Hub Sites
-  const scopingFilter: Filter<"content"> = {
-    filterType: "content",
-    type: "$site",
-  };
-  // merge filters
-  const sitesFilter = mergeContentFilter([scopingFilter, filter]);
-  // delegate
-  return searchContentEntities(sitesFilter, convertItemToSite, options);
+): Promise<IHubSearchResponse<IHubSite>> {
+  let qry: IQuery;
+
+  if (typeof query === "string") {
+    qry = createQueryFromString(query, "term", "item");
```

Comment:
Do we need the `term` argument here? As in, are we expecting any other filter to be applied as a string other than a general `term`?

Reply:
When searching for users, the generic thing may be `name`. I also considering enabling a query like `tag: water` which we could split and the use this same fn - or sniff for guid's and then use `id`. Viv built something like that in the early versions of the gallery and it was handy